tests/unittests/Foundation/NSPointerFunctionsTests.mm

```diff
@@ -80,7 +80,8 @@ NSUInteger NSPointerFunctionsTestStructSizeFunction(const void* item) {
     const void* ptr3 = (void*)((long)ptr1 + 1);
 
     testEqualFunction(functions, ptr1, ptr2, ptr3);
-    testHashFunction(functions, ptr1, ptr2, ptr3);
+    // Disabling hash function check here as performing hash on opaque values cannot be accuratly matched with the reference platform.
+    // testHashFunction(functions, ptr1, ptr2, ptr3);
```

Comment:
Fair enough with regard to value of testing. I'm still wondering whether we should change to a right shift to match the reference platform.

---
In reply to: [73765315](https://github.com/Microsoft/WinObjC/pull/833#discussion_r73765315) [](ancestors = 73765315,73764838,73764075,73762989)